week09_policy_II/ppo.ipynb

```diff
@@ -16,7 +16,7 @@
     "# os.system('pip install pyglet==1.2.4')\n",
     "# os.system('pip install gym')\n",
     "\n",
-    "# prefix = 'https://raw.githubusercontent.com/yandexdataschool/Practical_RL/spring19/week06_policy_II/'\n",
+    "# prefix = 'https://raw.githubusercontent.com/yandexdataschool/Practical_RL/master/week06_policy_II/'\n",
```

Comment:
This looks like a broken link.

Reply:
it is just a prefix, the subsequent code does stuff like 
```os.system('wget ' + prefix + 'runners.py')```

Reply:
yeah... you were right, it should be week09 :)